content/en/tracing/serverless_functions/_index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Trace Serverless Functions
+kind: documentation
+description: 'Send traces from your Serverless Functions to Datadog'
+further_reading:
+    - link: 'serverless'
+      text: 'Serverless Monitoring with Datadog'
+---
+{{< img src="tracing/serverless_functions/ServerlessDistributedTrace.png" alt="Trace Serverless Functions"  style="width:100%;">}}
+
+In hybrid architectures, Serverless functions can form an important core of requests your application processes. Including these functions in your Distributed Traces can be critical for detecting performance bottlenecks and service outages in your distributed systems.
+
+## Choose your Tracing Library
+
+Depending on your language and configuration, choose between using the Datadog Lambda Library or AWS X-Ray for your traces.
+
+{{< img src="tracing/serverless_functions/LambdaLibraryXRay.png" alt="Lambda Library or AWS X-Ray"  style="width:100%;">}}
+
+**Note:** If you are tracing across lambda functions and hosts, you will need to use **both** Datadog APM and AWS X-Ray.
+
+## Enable Datadog APM
```

Comment:
```suggestion
## Recommended: Enable Datadog APM
```

Reply:
@andrewardito can we do something like this to guide customers along the happy path? If I'm an engineer with two hours to set up APM on my service, I'm not sure I'd be able to figure this out.